src/scripts/main.js

```diff
@@ -1,3 +1,59 @@
 'use strict';
 
-// write code here
+const appendRow = document.querySelector('.append-row');
+const removeRow = document.querySelector('.remove-row');
+const appendColumn = document.querySelector('.append-column');
+const removeColumn = document.querySelector('.remove-column');
+
+const table = document.querySelector('tbody');
+const tableRow = document.querySelector('tr');
+const minElements = 3;
+const maxElements = 10;
+
+function addRow(e) {
+  table.append(tableRow.cloneNode('cloned'));
+  removeRow.disabled = false;
+
+  if (table.children.length === maxElements) {
+    appendRow.disabled = true;
+  }
+}
```

Comment:
```suggestion
function addRow() {
  table.append(tableRow.cloneNode('cloned'));
  removeRow.disabled = false;

  if (table.children.length === maxElements) {
    appendRow.disabled = true;
  }
}
```
не нужно принимать аргументы если ты их не используешь